pkg/codegen/docs/gen.go

```diff
@@ -1799,7 +1598,6 @@ func (mod *modContext) genIndex() indexData {
 		Functions:          functions,
 		Modules:            modules,
 		PackageDetails:     packageDetails,
-		LanguageLinks:      mod.getLanguageLinks(),
```

Comment:
The use of this in the template was removed in https://github.com/pulumi/pulumi/pull/6270.